squirrel.frontier/src/main/java/org/dice_research/squirrel/frontier/impl/FrontierImpl.java

```diff
@@ -246,63 +299,64 @@ public void addNewUri(CrawleableUri uri) {
                 }
                 knownUriFilter.add(uri, System.currentTimeMillis());
             } else {
-                LOGGER.warn("addNewUri(" + uri + "): " + uri.getUri().getScheme() + " is not supported, only " + schemeUriFilter.getSchemes() + ". Will not added!");
+                LOGGER.warn("addNewUri(" + uri + "): " + uri.getUri().getScheme() + " is not supported, only "
+                        + schemeUriFilter.getSchemes() + ". Will not added!");
             }
-            
+
         } else {
-            LOGGER.info("addNewUri(" + uri + "): URI is not good [" + knownUriFilter + "]. Will not be added!");
+            LOGGER.debug("addNewUri(" + uri + "): URI is not good [" + knownUriFilter + "]. Will not be added!");
         }
     }
 
+    public static String getDomainName(URI uri) throws URISyntaxException {
+        String domain = uri.getHost();
+        return domain.startsWith("www.") ? domain.substring(4) : domain;
```

Comment:
We had something similar in the Domain Based queue. 1) I am not sure whether we should remove `"www."` and 2) methods like that should be made static (which it is here) and moved to a general class which can be used by all other classes. This avoids the situation that we have now (in this method we are removing `"www."` while in the queue method, we keep it).

Reply:
I centralized all the method calls using one from UriUtils. We can discuss later if we should remove the www. or not